packages/shared/src/components/filters/AdvancedSettings.spec.tsx

```diff
@@ -27,14 +27,16 @@ import { getFeedSettingsQueryKey } from '../../hooks/useMutateFilters';
 import { waitForNock } from '../../../__tests__/helpers/utilities';
 
 const showLogin = jest.fn();
+// eslint-disable-next-line prefer-const, no-undef-init
```

Comment:
Why is this necessary? You don't actually need to initialize it with `undefined` because this is the default value.
And also eslint should not complain about `let` if you actually use it.

Reply:
Ah, I see, that could be why it was throwing an error.